tools/lib/template_parser.py

```diff
@@ -320,6 +320,7 @@ def is_special_html_tag(s: str, tag: str) -> bool:
 
 
 OPTIONAL_CLOSING_TAGS = [
+    "br",
```

Comment:
Does this belong in a previous commit?  I think one of the first commits I saw added a `br /` somewhere, but wasn't looking too closely.

Reply:
Yes, fixed. The context is that the FormatJS parser requires the XML-style syntax for a self-closing tag.